test.js

```diff
@@ -1,22 +1,16 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var taskkill = require('./');
+import childProcess from 'child_process';
+import test from 'ava';
+import fn from './';
```

Comment:
I've decided to use `m` (for module) instead of `fn`, as the export isn't always a function.